src/swarm/Const.d

```diff
@@ -109,7 +109,7 @@ public struct NodeItem
 
```

Comment:
Is the commit message right (`Since HEAD is currently using v5.x.x`)?

Reply:
No you're right. Fixed.